src/layout-rect.js

```diff
@@ -55,7 +55,7 @@ export function layoutRectLtwh(left, top, width, height) {
 /**
  * Creates a layout rect based on the DOMRect, e.g. obtained from calling
  * getBoundingClientRect.
- * @param {!Rect} rect
+ * @param {!ClientRect} rect
```

Comment:
it should be `DOMRect` i think the closure externs are out of date. will research and file bug if needed on their end